server/audit_math/ballot_polling.py

```diff
@@ -19,6 +19,7 @@ def get_sample_size(
     risk_limit: int,
     contest: Contest,
     sample_results: Optional[Dict[str, Dict[str, int]]],
+    round_sizes: Dict[int, int],
```

Comment:
It seems more intuitive to me to pass the round sizes before the sample results (mirroring the sequence of events in the audit procedure) but of course it matters little.